pkg/reconciler/instances/istio/actions/performer_test.go

```diff
@@ -292,6 +292,29 @@ func Test_DefaultIstioPerformer_PatchMutatingWebhook(t *testing.T) {
 		require.Contains(t, gotNew.Webhooks[0].NamespaceSelector.MatchExpressions, want)
 		require.NotContains(t, gotOld.Webhooks[0].NamespaceSelector.MatchExpressions, want)
 	})
+
+	t.Run("patching mutating webhook should be idempotent", func(t *testing.T) {
+		// given
+		whConfName := "istio-revision-tag-default"
+		kubeClient := mocks.Client{}
+		mutatingWebhookConf := createIstioMutatingWebhookConf(whConfName)
+		clientset := fake.NewSimpleClientset(mutatingWebhookConf)
+		kubeClient.On("Clientset").Return(clientset, nil)
+		wrapper := NewDefaultIstioPerformer(nil, nil, nil)
+
+		// when
+		err := wrapper.PatchMutatingWebhook(context.TODO(), &kubeClient, log)
+		require.NoError(t, err)
```

Comment:
I would add first `// then` clause here.

Reply:
isn't the convention to execute testee object with when? I'll add more explicit comment instead.